### ISSUE TITLE
allow GenotypeGVCFs to perform combination of multiple gvcf files

### ISSUE THREAD

**Issue by lbergelson** [broadinstitute/gatk#2584]

**Comment by droazen** [comment]
One very simple way to do this would be for `GenotypeGVCFs` to simply invoke `CombineGVCFs` directly upon initialization when multiple inputs are specified. We should consider doing this as a quick way of restoring this functionality, as several users have requested this.

**Comment by sooheelee** [comment]
Yes, one such user's request is at https://gatkforums.broadinstitute.org/gatk/discussion/comment/47849#Comment_47849.
